spec/polaris-management-service.yml

```diff
@@ -938,6 +940,34 @@ components:
           format: password
           description: Bearer token (input-only)
 
+    SigV4AuthenticationParameters:
+      type: object
+      description: AWS Signature Version 4 authentication
+      allOf:
+        - $ref: '#/components/schemas/AuthenticationParameters'
+      properties:
+        roleArn:
+          type: string
+          description: The aws IAM role arn assume when signing requests
+          example: "arn:aws:iam::123456789001:role/role-that-has-remote-catalog-access"
+        externalId:
+          type: string
+          description: An optional external id used to establish a trust relationship with AWS in the trust policy
+        signingRegion:
+          type: string
+          description: Region to be used by the SigV4 protocol for signing requests
```

Comment:
does this region needs to be same as that of Polaris is running is Sigv4 suffciient for cross region or we need SigV4A ?

Reply:
I see, my question is a bit different, do we need the assertion that AWS region of polaris service and signing region is same ? or do current SigV4 is can handle cross regions.

Reply:
Yeah, signing region is mandatory. If it's not explicitly provided, we can infer it based on the region Polaris is running in. I'm also totally fine with making it a required field to keep things clear and explicit.

Reply:
Hey @singhpk234, Polaris might be running in region A, while the remote catalog (like Glue) could be in region B.

Reply:
from my POV explicit region is preferable to assuming the region where Polaris runs (the latter may not be even inside AWS infra). Defaulting to the env.-level region setting is fine, though.

Reply:
Done!